SqlFundamentalsDemo/SqlDemo/trg_InsertEmployee.sql

```diff
@@ -0,0 +1,21 @@
+﻿CREATE TRIGGER [trg_InsertEmployee]
+	ON [dbo].[Employee]
```

Comment:
I would say there is an extra tabulation for all items after 1st row, let's remove it